src/SDK/SDK.php

```diff
@@ -634,6 +637,27 @@ public function generate(string $target): void
                         }
                     }
                     break;
+                case 'enum':
+                    foreach ($this->spec->getServices() as $key => $service) {
+                        $methods = $this->spec->getMethods($key);
+
+                        foreach ($methods as $method) {
+                            $params['method'] = $method;
+                            $parameters = $method['parameters']['path'];
+
+                            foreach ($parameters as $parameter) {
+                                // Check if the enum field is defined
+                                if ($parameter['enum'] != null) {
+                                    $params['enum'] = [
+                                        'name' =>  $parameter['enum-name'],
```

Comment:
Isn't this supposed to be `x-enum-name` ? @2002Bishwajeet

Reply:
I did this mapping [here](https://github.com/appwrite/sdk-generator/blob/fde85780a860a278e7db3b17b76827b0f7e2863c/src/Spec/Swagger2.php#L231) so hence the `enum-name`